locales/cy/translations.json

```diff
@@ -29,6 +29,12 @@
         "nameEnding" : "WELSH - Which private fund limited partnership name ending would you prefer to go on the public record?"
     },
 
+    "privateNameScotlandPage" : {
```

Comment:
the title uses 'Scottish', would it be more consistent to have privateNameScottishPage as the object/group name

Reply:
Yep, you're right. Will change this.

Reply:
Done.